docs/physics/montecarlo/initialization.ipynb

```diff
@@ -71,10 +71,39 @@
   },
```

Comment:
<p>This looks really good. One thing that would be great to show in this is perhaps the compilation time penalty for the first time you run this. Of course, you can only show this once in the notebook, but even comparing the first and second time you run the timing would show it (you may need to lower the number runs/loops)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>